src/plot_api/plot_api.js

```diff
@@ -2274,6 +2274,8 @@ Plotly.animate = function(gd, frameOrGroupNameOrFrameList, animationOpts) {
             var newFrame = trans._currentFrame = trans._frameQueue.shift();
 
             if(newFrame) {
+                gd.emit('plotly_animatingframe', newFrame.name);
```

Comment:
Making the event data an object seems like the way to go in case we want to add more stuff in their at some point.